lib/tasks/gettext.rake

```diff
@@ -7,16 +7,13 @@ gettext_find_task = begin
 if gettext_find_task
   namespace :gettext do
     task :store_action_names => :environment do
-      storage_file = "#{locale_path}/action_names.rb"
-      puts "writing action translations to: #{storage_file}"
-
-      File.write storage_file,
-                 "# Autogenerated!\n" +
-                 Actions::EntryAction
-                 .all_action_names
-                 .uniq
-                 .map { |n| %[_("#{n}")] }
-                 .join("\n")
+      # not relevant when extracting plugins
```

Comment:
This is not valid: it's relevant for plugins, that define actions for foreman-tasks (see https://github.com/Katello/katello/blob/master/locale/action_names.rb),

Better approach would be first introduce a mechanism that plugins that use the actions would use for extracting just names of their actions, and then removing this code completely. Without doing this, this PR just introduces regression IMO